torch/csrc/CudaIPCTypes.h

```diff
@@ -73,6 +53,7 @@ namespace {
 
 constexpr int64_t CUDA_IPC_REF_COUNTER_FILE_SIZE = 10000;
 constexpr int64_t CUDA_IPC_WARN_AFTER_X_BLOCKS_IN_LIMBO = 1000;
+constexpr int64_t CUDA_IPC_MAXIMUM_EVENTS_TO_USE = 1000;
```

Comment:
How did we determine this number, and how much leeway do we have?

Reply:
There is no documentation about this OOM. Test models we have available shares around 100-200 tensors, that is where 1000 comes from. However I don't want this number to be too big as allocating over 20,000 MP events raises exception.

Reply:
Yeah, my point is, a little comment saying, "This was determined empirically that ... and to give us leeway, we picked ..." Additionally, saying which version of CUDA you did the empirical test on will also help later people who need to update the constant.